client/blocks/hello-vote-notice/index.jsx

```diff
@@ -70,6 +72,10 @@ export default connect(
 				isSiteSection( state ) &&
 				'settings' !== getSectionName( state ) &&
 				canCurrentUser( state, selectedSiteId, 'manage_options' ) &&
+				(
+					! userStartDate ||
+					moment( userStartDate ).isBefore( moment().subtract( 1, 'weeks' ) )
+				) &&
```

Comment:
>  After we polish a few things, we'd like to organize our Guided Tours files a little better,

Though, for such a specific function as `isNewUser`, we could just move it to `state/current-user` and have Guided Tours require it from there.